lib/internal/Magento/Framework/DB/AbstractMapper.php

```diff
@@ -392,9 +393,10 @@ protected function getConditionFieldName($fieldName)
 
     /**
      * Hook for operations before rendering filters
+     *
      * @return void
      */
-    protected function renderFiltersBefore()
+    protected function renderFiltersBefore() //phpcs:ignore Magento2.CodeAnalysis.EmptyBlock
```

Comment:
I'd go adding this to the docs block section.

Reply:
@andrewbess Please pay attention

Reply:
@eduard13 @Den4ik 
It doesn't work when it is added in doc block

Reply:
@andrewbess What about `phpcs:disable`?